modules/serverless-troubleshooting-monitoring-and-logging.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * serverless/reference/troubleshooting.adoc
+
+:_content-type: REFERENCE
+[id="serverless-troubleshooting-monitoring-and-logging_{context}"]
+= Monitoring and logging
+
+*How can I access logs of my {ServerlessProductShortName} containers?*
+
+Container logs are available using the integration with {product-title} logging.
```

Comment:
I suppose after reading this sentence the user is supposed to look at this doc? https://github.com/openshift/openshift-docs/pull/45262/files#diff-17323afbc27eee94cd231ac0ad14a1d1b8c2278b059388b45db3db0d6d438fddR47 
It's not really clear where the user should go from here.